pure3.md

```diff
@@ -0,0 +1,273 @@
+# PURE-3
+
+* PURE: 3
+* Title: Scudo: Adding Supply Chain Security Capabilities to Uptane
+* Version: 1
+* Last-Modified: 2022-11-02
+* Authors: Marina Moore <marinamoore@nyu.edu>, Aditya Sirish <aditya.sirish@nyu.edu>, Lois Anne Delong <lad278@nyu.edu>
+* Status: Draft
+* Content-Type: <text/markdown>
+* Created: 2022-03-30
+
+## Abstract
+
+Software supply chain attacks are on the rise. A 2022 Sonatype report found a
+742% increase in the last three years. As a result, companies across all
+industry sectors have turned their attention to software supply chain security. 
+The need to find solutions quickly is further fueled by rapidly increasing
+regulations and standards, including President Biden’s U.S.Executive Order
+14028.
+
+In this PURE, we propose Scudo, a framework that combines the compromise
+resilience and secure delivery protection of Uptane and the end-to-end software
+supply chain security guarantees offered by in-toto. It describes how in-toto
+metadata can be integrated into an automotive software supply chain that already
+uses Uptane. It further establishes verification responsibilities for ECUs based
+on the resources they possess and presents a stop-gap option for automobiles
+with no sufficiently capable ECUs.
+
+## Motivation
+
+Software supply chain attacks are on the rise with the
+[2022 State of the Software Supply Chain report](https://www.sonatype.com/state-of-the-software-supply-chain/introduction),
+noting a 742% increase over the course of the last three years. But, it was the
+[2020 Sunburst attack](https://www.cynet.com/attack-techniques-hands-on/sunburst-backdoor-c2-communication-protocol/)
+that really made both government and corporate CISOs take action. Affecting more
+than 100 companies and nine government agencies, the cause of the attack was
+Sunburst malware that had been uploaded via a routine update from Orion software
+produced by SolarWinds. The attack taught both public and private concerns some
+hard lessons, such as:
+
+* The very thing designed to protect the security of software-patches and
+  updates that fix routine vulnerabilities-can also be used to deliver malware
+  that can damage or corrupt.
+* As software products contain third-party components, program developers are
+  not in complete control of the contents of their products.
+* Malware can linger undetected for extended periods of time, allowing it to
+  infect every system that touches it. The Sunburst malicious code was part of
+  an update released by SolarWinds in the spring of 2020, yet it was not
+  reported until December of that year.
+
+Automotive software supply chains are particularly tempting targets for
+attackers. This is because automotive software runs on a large number of
+vehicles and updating all of them to recover quickly from an attack is a
```

Comment:
I was alluding to a process issue more than delivering the update itself. Getting thousands of vehicles all patched is different from, say, smartphones. Let me try to rephrase.

Reply:
```
Automotive software supply chains are particularly tempting targets for
attackers. This is because automotive software runs on a large number of
vehicles and updating all of them to recover quickly from an attack is a
slow and difficult proposition. As such, a successful software supply chain
attack means that thousands (or more) of vehicles are impacted. Any attack that
targets a large number of vehicles on roads can do immeasurable damage. Further,
while Uptane can securely deliver updates to vehicles to recover from the
attack, ensuring every vehicle on the road receives the update immediately is
very difficult given the circumstances of use of these vehicles. Therefore,
strategies that can secure automotive software supply chains are vital to
_avoid_ such attacks altogether.
```